### ISSUE TITLE
Class 'Devfactory\Minify\MinifyServiceProvider' not found

### ISSUE THREAD

**Issue by GilEyni** [DevFactoryCH/minify#25]
Hi

I'm sure it's my fault but I can't seem to figure it out.

Added in composer.json: 

```
"require": {
        "laravel/framework": "5.0.*",
        "devfactory/minify": "1.0.*"
    },
```

Run composer update and got : "Installing devfactory/minify (1.0.7)"

Added in config/app.php:

`'Devfactory\Minify\MinifyServiceProvider',`

But getting: Class 'Devfactory\Minify\MinifyServiceProvider' not found

**Comment by sirsquall** [comment]
Strange have you try to run  `composer dump-autoload`?

**Comment by RickBakkr** [comment]
Same problem over here.

Running `composer dump-autoload` won't solve the issue.

**Comment by RickBakkr** [comment]
Just got it solved!

I first ran `composer install`, apparently composer had not downloaded your package inside the vendor folder. I then ran `composer update` and the devfactory folder did exist in the vendor folder.
Then run `php artisan cache:clear`, to clear the cache for Laravel to work again.

So, @GilEyni if I were you, I would check the vendor directory for the existance of a child folder named devfactory, if it doesn't, try above :)
